docs/literate/graphs/graphs.jl

```diff
@@ -0,0 +1,325 @@
+using Catlab, Catlab.Theories
+using Catlab.CategoricalAlgebra
+using Catlab.Graphs
+using Catlab.Graphics
+using Catlab.Graphics.Graphviz
+#import Catlab.Graphics.Graphviz: to_graphviz, to_graphviz_property_graph
+
+using Colors
+draw(g) = to_graphviz(g, node_labels=true, edge_labels=true)
+
+GraphvizGraphs.to_graphviz(f::ACSetTransformation; kw...) =
+  to_graphviz(GraphvizGraphs.to_graphviz_property_graph(f; kw...))
+
+function GraphvizGraphs.to_graphviz_property_graph(f::ACSetTransformation; kw...)
```

Comment:
This is nice. We should upstream something like it to `Catlab.Graphs`.

Reply:
Yeah for sure, I'll post it in an issue.